packages/neuron-wallet/src/entities/Transaction.ts

```diff
@@ -44,9 +44,9 @@ export default class Transaction extends BaseEntity {
   outputs!: Cell[]
 
   @Column({
-    type: 'bigint',
+    type: 'varchar',
```

Comment:
`string` will be cast to `varchar` automatically

Reply:
Different database will have different behave, like number to int or float? I don't want to depend on default type cast